flac.md

```diff
@@ -80,29 +80,29 @@ All numbers used in a FLAC bitstream are integers; there are no floating-point r
-- [FLAC](#flac-subset) specifies a subset of itself as the Subset format. The purpose of this is to ensure that any streams encoded according to the Subset are truly "streamable", meaning that a decoder that cannot seek within the stream can still pick up in the middle of the stream and start decoding. It also makes hardware decoder implementations more practical by limiting the encoding parameters such that decoder buffer sizes and other resource requirements can be easily determined. [flac]{.commandname} generates Subset streams by default unless the "--lax" command-line option is used. The Subset makes the following limitations on what may be used in the stream:
-  - The blocksize bits in the [frame header](#frame_header) must be 0001-1110. The blocksize must be <= 16384; if the sample rate is <= 48000 Hz, the blocksize must be <= 4608.
-  - The sample rate bits in the [frame header](#frame_header) must be 0001-1110.
-  - The bits-per-sample bits in the [frame header](#frame_header) must be 001-111.
-  - If the sample rate is <= 48000 Hz, the filter order in [LPC subframes](#subframe_lpc) must be less than or equal to 12, i.e. the subframe type bits in the [subframe header](#subframe_header) may not be 101100-111111.
-  - The Rice partition order in a [Rice-coded residual section](#partitioned_rice) must be less than or equal to 8.
+- `FLAC` specifies a subset of itself as the Subset format. The purpose of this is to ensure that any streams encoded according to the Subset are truly "streamable", meaning that a decoder that cannot seek within the stream can still pick up in the middle of the stream and start decoding. It also makes hardware decoder implementations more practical by limiting the encoding parameters such that decoder buffer sizes and other resource requirements can be easily determined. [flac]{.commandname} generates Subset streams by default unless the "--lax" command-line option is used. The Subset makes the following limitations on what may be used in the stream:
+  - The blocksize bits in the [frame header](#frameheader) must be 0001-1110. The blocksize must be <= 16384; if the sample rate is <= 48000 Hz, the blocksize must be <= 4608.
+  - The sample rate bits in the [frame header](#frameheader) must be 0001-1110.
+  - The bits-per-sample bits in the [frame header](#frameheader) must be 001-111.
+  - If the sample rate is <= 48000 Hz, the filter order in [LPC subframes](#subframe_lpc) must be less than or equal to 12, i.e. the subframe type bits in the [subframe header](#subframeheader) may not be 101100-111111.
+  - The Rice partition order in a [Rice-coded residual section](#partitionedrice) must be less than or equal to 8.
```

Comment:
I think that I was inaccurate in my original anchor - I think this should be
`[Rice-coded residual section](#residualcodingmethodpartitionedrice)`